integrations/slack/api/post.go

```diff
@@ -38,9 +38,7 @@ const (
 // slackURL is a var and not a const for unit-testing purposes.
 var slackURL = "https://slack.com/api/"
 
-type ctxKey string
-
-var OAuthTokenContextKey = ctxKey("OAuthTokenContext")
+var OAuthTokenContextKey = struct{}{}
```

Comment:
return to how it was before

Reply:
Done.

I researched this as we discussed offline, and I still prefer `struct{}` which AFAIK is the official recommendation of the Go team, and more efficient memory-wise.

Anyway, we all agree that `type ctxKey string` is not as terrible as a regular string, so it's good enough.

Reply:
@tebeka?

Reply:
probelm is that struct{}{} is equal to any other struct{}{}.

Reply:
You could say the same thing about a type alias for strings: what if I define a different type in a different package, but use the same string as a key?

Anyway, this is interesting topic. I'll follow-up offline with Miki to see his sources, either way I'll learn something new. :)